README.md

```diff
@@ -26,6 +26,7 @@ over 30,000 e-mail with lots of huge attachments stored across over 450 folders.
   * Flagged
   * Deleted
   * Draft
+* Optionally preserve user-defined flags (also known as "IMAP keywords" or "tags")
```

Comment:
please add link to https://tools.ietf.org/html/rfc5788 in case users don't know about IMAP keywords/flags